controllers/ptpoperatorconfig_controller.go

```diff
@@ -198,6 +200,7 @@ func (r *PtpOperatorConfigReconciler) syncLinuxptpDaemon(ctx context.Context, de
 	data.Data["KubeRbacProxy"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
 	data.Data["SideCar"] = os.Getenv("SIDECAR_EVENT_IMAGE")
 	data.Data["NodeName"] = os.Getenv("NODE_NAME")
+	data.Data["TransportStorageType"] = DefaultTransportStorageType
```

Comment:
storageType ( can you use webook validation ?)

Reply:
done